src/apps/Hosting/Files.js

```diff
@@ -155,7 +161,14 @@ const HostingFilesView = React.createClass({
         >
           <Show if={items.length && !isLoading}>
             <RaisedButton
```

Comment:
put these buttons in a flex div. they are both `inline-block`s and they are not aligned properly due to `<Show>` component. I find that better than adding a vertical align style to every single button inside.